gamemodes/terrortown/gamemode/client/cl_help.lua

```diff
@@ -409,6 +409,9 @@ function HELPSCRN:CreateInterfaceSettings(parent)
 	cb = form:CheckBox(GetTranslation("disable_overheadicons"), "ttt2_disable_overheadicons")
 	cb:SetTooltip(GetTranslation("disable_overheadicons_tip"))
 
+	cb = form:CheckBox(GetTranslation("disable_doubletap_sprint"), "ttt2_disable_doubletap_sprint")
+	cb:SetTooltip(GetTranslation("disable_doubletap_sprint_tip"))
```

Comment:
This should be moved to the gameplay form, since it has nothing to do with the interface

Reply:
Ha, the same thing I recommended beforehand :D